kwave/utils/mapgen.py

```diff
@@ -259,97 +239,77 @@ def hounsfield2density(ct_data, plot_fitting=False):
     return density
 
 
-def water_sound_speed(temp):
+def water_sound_speed(temp: float) -> float:
```

Comment:
Not sure if this is related to `mapgen`.

Reply:
I think this is arguable. I will leave it here for now. Perhaps we can split off a propgen in the future?

Reply:
Sounds good!